### DISCUSSION TITLE
use Cached credentials for sls deploy command

### DISCUSSION THREAD

**Discussion by s1mrankaur** [serverless/serverless#9264] [Q&A]
I am caching my credentials with aws-vault but sls deploy still prompts for the MFA code. Is there a way to use the cached credentials instead of being prompted for MFA code on every deploy?

**Comment by mikayla95** [comment]
have you looked into using an sts token? That's what I had to do to bypass MFA

**Comment by pgrzesik** [comment]
Hello @s1mrankaur - as this is more of a support question, I'm moving it to "Discussions" - this tracker is dedicated to feature requests and/or bugs reports.

**Comment by garethmcc** [comment]
@s1mrankaur Are you attempting to store credentials for a headless system? You could instead use the provide access key feature if that makes things easier: https://www.serverless.com/framework/docs/guides/cicd/running-in-your-own-cicd#create-an-access-key-in-the-serverless-framework-dashboard
